mne_bids/tests/test_mne_bids.py

```diff
@@ -78,10 +80,25 @@ def test_fif():
         write_raw_bids(raw, er_bids_basename_bad, output_path, overwrite=False)
 
     # give the raw object some fake participant data
-    raw = mne.io.read_raw_fif(raw_fname)
-    raw.anonymize()
     raw.info['subject_info'] = {'his_id': subject_id2,
                                 'birthday': (1994, 1, 26), 'sex': 1}
+    write_raw_bids(raw, bids_basename, output_path, events_data=events_fname,
+                   event_id=event_id, overwrite=True)
+    # assert age of participant is correct
+    participants_tsv = op.join(output_path, 'participants.tsv')
+    data = _from_tsv(participants_tsv)
+    assert data['age'][data['participant_id'].index('sub-01')] == '8'
+
+    # try and write preloaded data
+    raw = mne.io.read_raw_fif(raw_fname, preload=True)
```

Comment:
this is not tested elsewhere? I would be surprised if not ...

Reply:
well the coverage report said it wasn't covered so I guess not?